R/tweak-page.R

```diff
@@ -47,10 +47,16 @@ tweak_rmarkdown_html <- function(html, input_path, pkg = list(bs_version = 3)) {
   src <- xml2::xml_attr(img, "src")
   abs_src <- is_absolute_path(src)
   if (any(abs_src)) {
+    img_target_nodes <- img[abs_src]
+    img_src_real <- path_real(URLdecode(src[abs_src]))
```

Comment:
Thanks for tackling this, but would you mind looking around for an alternative encoding/decoding function? (I feel like there has to be one in pkgdown's existing dependencies). The base functions are cripplingly slow. 

(Also no need to import these functions; just use with `::`.

Reply:
Using now `{xml2}` functions.